localstack/services/awslambda/invocation/docker_runtime_executor.py

```diff
@@ -336,6 +338,8 @@ def start(self, env_vars: dict[str, str]) -> None:
             for source, target in container_config.copy_folders:
                 CONTAINER_CLIENT.copy_into_container(self.container_name, source, target)
 
+        end_time = time.perf_counter()
+        usage.initduration.record_value(end_time - start_time)
```

Comment:
I guess that's the easiest way to reliably measure container initialization overhead. Terminology-wise, this is something completely different compared to what AWS refers to `INIT` https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtime-environment.html

In the config, we use `LAMBDA_RUNTIME_ENVIRONMENT_TIMEOUT`. Potential alternatives are `container_init_duration` or more generic `env_init_duration`.

I understand `initduration`; just wanted to mention the different meaning compared to AWS terminology so we are aware of it.

Reply:
Actually that should be removed right now, since I also removed the `initduration` counter in a previous commit :grimacing: 

But your point is still valid :+1:  We can think about adding this back at a later date